### ISSUE TITLE
Consider adding a TOPP-RA example

### ISSUE THREAD

**Issue by sea-bass** [sea-bass/pyroboplan#55]
https://github.com/hungpham2511/toppra has some examples we can use along with this library.

**Comment by Skwarson96** [comment]
Hello
I've launched  [toppra](https://github.com/hungpham2511/toppra) repository. Do you have any specific idea in mind on how to use this repository in pyroboplan project?

**Comment by sea-bass** [comment]
Yes -- the idea would be to take a planned path, e.g., from RRT or Cartesian Planning, for which there are examples, and then retime it to be time-optimal using TOPP-RA.

**Comment by Skwarson96** [comment]
Ok, then I'll try to add something like this 😃
